programs/ssl/ssl_server2.c

```diff
@@ -3742,6 +3745,12 @@ int main( int argc, char *argv[] )
     }
 #endif /* MBEDTLS_SSL_DTLS_CONNECTION_ID */
 
+#if defined(MBEDTLS_MEMORY_DEBUG)
+    mbedtls_memory_buffer_alloc_cur_get(&current_heap_memory, &heap_blocks);
+    mbedtls_memory_buffer_alloc_max_get(&peak_heap_memory, &heap_blocks);
+    mbedtls_printf("Heap memory usage after handshake: %lu bytes. Peak memory usage was %lu\n", current_heap_memory, peak_heap_memory);
```

Comment:
You'll need to cast to an `unsigned long` here to avoid warnings with 64-bit MSVC, I'd expect.

Reply:
Thanks for that attention. I added casting